heartbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,261 @@
+
+. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon
+and select *Run As Administrator*). If you are running Windows XP, you may need
+to download and install PowerShell.
+
+. Run the following commands to install Heartbeat as a Windows service:
++
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\Heartbeat'
+PS C:\Program Files\Heartbeat> .\install-service-heartbeat.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system, you need to set the
+execution policy for the current session to allow the script to run. For
+example:
++PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-heartbeat.ps1+.
+
+Before starting Heartbeat, you should look at the configuration options in
+the configuration file, for example +C:\Program Files\Heartbeat\heartbeat.yml+
+or +/etc/heartbeat/heartbeat.yml+. For more information about these
+options, see <<heartbeat-configuration-details>>.
+
+[[heartbeat-configuration]]
+=== Step 2: Configuring Heartbeat
+
+To configure Heartbeat, you edit the configuration file. For rpm and deb,
+you'll find the configuration file at +/etc/heartbeat/heartbeat.yml+.
+For mac and win, look in the archive that you just extracted.
+
+Heartbeat uses monitors to configure ping behavior. You configure each
+monitor individually. Heartbeat currently provides monitors for ICMP ping,
+TCP ping, and HTTP-based ping (see <<heartbeat-overview>> for more about these
+monitors). Here is an example that configures Heartbeat to use an `icmp`
+monitor:
+
+[source,yaml]
+----------------------------------------------------------------------
+heartbeat.monitors:
+- type: icmp 
+  name: icmp
+  enabled: true
+  schedule: '*/5 * * * * * *' 
+  hosts: ["myhost"]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+  wait: 1s
+heartbeat.scheduler:
+  location: 'UTC-08:00'
+output.elasticsearch:
+  hosts: ["myhost:9200"]
+----------------------------------------------------------------------
+
+To configure Heartbeat:
+
+. Specify the list of monitors that you want to enable. Each item in the list
+begins with a dash (-). The following example configures Heartbeat to use two
+monitors, an `icmp` monitor and a `tcp` monitor:
++
+[source,yaml]
+----------------------------------------------------------------------
+heartbeat.monitors:
+- type: icmp
+  name: icmp
+  enabled: true
+  schedule: '*/5 * * * * * *' <1>
+  hosts: ["myhost"]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+  wait: 1s
+- type: tcp 
+  name: tcp
+  enabled: true
+  schedule: '@every 5s' <2>
+  hosts: ["myhost:12345"]
+  ipv4: true
+  ipv6: true
+  mode: any <3>
+  timeout: 16s
+heartbeat.scheduler:
+  location: 'UTC-08:00' <4>
+output.elasticsearch:
+  hosts: ["myhost:9200"]
+----------------------------------------------------------------------
```

Comment:
for getting started only I wouldn't show `heartbeat.schedule` section at all. It's a more advanced setting.